resources/views/users/view.blade.php

```diff
@@ -293,7 +293,7 @@
 
               @can('view', $user)
                 <div class="col-md-12" style="padding-top: 5px;">
-                  <a href="{{ route('users.print', $user->id) }}" style="width: 100%;" class="btn btn-sm btn-default hidden-print">{{ trans('admin/users/general.print_assigned') }}</a>
+                  <a href="{{ route('users.print', $user->id) }}" style="width: 100%;" class="btn btn-sm btn-default hidden-print target=_blank rel=noopener">{{ trans('admin/users/general.print_assigned') }}</a>
```

Comment:
Looks like some quote marks were off here... Should be:

`class="btn btn-sm btn-default hidden-print" target="_blank" rel="noopener">`

It's alright tho - I accepted it and am fixing it now.